tests/client-dbus/tests/udev/test_udev.py

```diff
@@ -218,7 +223,9 @@ def start_service(self):
         if service.poll() is not None:
             raise Exception("Daemon unexpectedly exited with %s" % service.returncode)
```

Comment:
Might as well change the ```raise Exception``` here to ```raise RuntimeError``` also.

Reply:
Resolved.